backend/census_historical_migration/base_field_maps.py

```diff
@@ -0,0 +1,17 @@
+from collections import namedtuple as NT
+
+FormFieldInDissem = WorkbookFieldInDissem = 1000
+
+# This provides a way to map the sheet in the workbook to the
+# column in the DB. It also has a default value and
+# the type of value, so that things can be set correctly
+# before filling in the XLSX workbooks.
+# Define a new named tuple with an additional field
+SheetFieldMap = NT("SheetFieldMap", "in_sheet in_db in_dissem default type")

```

Comment:
Strictly as a comment, not a proposed revision:

These are currently named tuples, or "records." If turning these into classes would make them more useful, it is completely reasonable to do so. FYI.

Reply:
Agreed, actually I took that approach in my initial commit but I can remember why I diverted path.